lib/src/test/kotlin/network/provider/ProviderTest.kt

```diff
@@ -268,31 +262,11 @@ class ProviderTest {
         assertEquals(PriceUnit.WEI, receipt.actualFee.unit)
     }
 
-    @Test
-    fun `get declare v1 transaction`() {
-        assumeTrue(NetworkConfig.isTestEnabled(requiresGas = false))
-
-        val transactionHash = declareV1TransactionHash
-        val tx = provider.getTransaction(transactionHash).send() as DeclareTransactionV1
-        assertNotEquals(Felt.ZERO, tx.classHash)
-        assertEquals(transactionHash, tx.hash)
-
-        val receiptRequest = provider.getTransactionReceipt(transactionHash)
-        val receipt = receiptRequest.send()
-
-        assertTrue(receipt is DeclareTransactionReceipt)
-        assertFalse(receipt.isPending)
-        assertTrue(receipt.isAccepted)
-        assertEquals(TransactionExecutionStatus.SUCCEEDED, receipt.executionStatus)
-        assertEquals(TransactionFinalityStatus.ACCEPTED_ON_L1, receipt.finalityStatus)
-        assertNull(receipt.revertReason)
-    }
-
```

Comment:
We want remove the ability for user to send the declare v1 requests, but they still should be able to get the v1 transactions from the network.